tools/logicalimager/tsk_logical_imager.cpp

```diff
@@ -917,8 +914,10 @@ main(int argc, char **argv1)
 
     if (iFlagUsed) {
         imgPaths.push_back(imgPath);
+        printDebug("Process is not running in elevated mode", NULL);
```

Comment:
Is this statement accurate? Using -i seems unrelated to admin mode

Reply:
I will remove it.